### ISSUE TITLE
'Custom Perspective' appears to do nothing in 6.0

### ISSUE THREAD

**Issue by codebling** [dbeaver/dbeaver#5900]
No window appears, as it did in 5.3. Tested in Windows + Mac. [Log](https://github.com/dbeaver/dbeaver/files/3175781/log.log)

```
org.eclipse.core.commands.ExecutionException: Error executing 'org.eclipse.ui.window.customizePerspective': java.lang.NullPointerException
	at org.eclipse.e4.core.commands.internal.HandlerServiceHandler.execute(HandlerServiceHandler.java:170)
[...]
Caused by: org.eclipse.e4.core.di.InjectionException: java.lang.NullPointerException
	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:68)

[...]
Caused by: java.lang.NullPointerException
	at org.jkiss.dbeaver.ui.actions.datasource.SelectActiveDataSourceHandler.updateElement(SelectActiveDataSourceHandler.java:98)
[...]
```

**Comment by dimipet** [comment]
same for me

**Comment by dimipet** [comment]
same for me

Eclipse IDE for Enterprise Java Developers.
Version: 2019-03 (4.11.0)
Build id: 20190314-1200

**Comment by uslss** [comment]
Fixed in 6.2.3
